test/TritonGPU/loop-pipeline-hip.mlir

```diff
@@ -263,3 +263,102 @@ module attributes {"triton_gpu.num-ctas" = 1 : i32, "triton_gpu.num-warps" = 4 :
     tt.return
   }
 }
+
+// -----
+
+// CHECK-LABEL:  tt.func @matmul_loop_fp8
```

Comment:
Do we really need to check the whole IR snippet? Just focusing on the important bits are fine?

Reply:
If we want to check that fp8 matmul loop is formed correctly, similar to fp16, I think the whole IR is important. However, there exist such test for fp16 loop-pipeline.mlir::matmul_loop, and I have tests for fpToFp folding (canonicalize.mlir) which caused issues for fp8 pipelining so maybe this test is redundant altogether. Let me know what you think.